types/nightwatch/index.d.ts

```diff
@@ -15,21 +15,21 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 4.5
 
-import { WebElement, WebElementPromise, By } from 'selenium-webdriver';
+import { WebElement, WebElementPromise, By, RelativeBy } from 'selenium-webdriver';
 
 export * from './globals';
 
 export const ELEMENT_KEY = 'element-6066-11e4-a52e-4f735466cecf';
 
 export interface JSON_WEB_OBJECT {
-    ELEMENT_KEY: string;
+    [ELEMENT_KEY]: string;
     getID: () => string;
 }
 
-export type definition = string | ElementProperties | Element;
+export type definition = string | ElementProperties | Element | RelativeBy;
 
 // TODO: Remove after Selenium-Webdriver implements locateWith in their types
-export function locateWith(args: By): any;
+// export function locateWith(args: By): any;
```

Comment:
please remove, do not left dead comment (condition's been satisfied as per comment)

Reply:
Thank you for requesting changes 👍. I'm still working on it, and I'd go through it again to remove unnecessary changes like dead comments, dead code, etc., after addressing review changes.